scene/resources/visual_shader_nodes.h

```diff
@@ -1995,4 +1995,43 @@ VARIANT_ENUM_CAST(VisualShaderNodeCompare::ComparisonType)
 VARIANT_ENUM_CAST(VisualShaderNodeCompare::Function)
 VARIANT_ENUM_CAST(VisualShaderNodeCompare::Condition)
 
+class VisualShaderNodeMultiplyAdd : public VisualShaderNode {
+	GDCLASS(VisualShaderNodeMultiplyAdd, VisualShaderNode);
+
+public:
+	enum Type {
+		TYPE_SCALAR,
+		TYPE_VECTOR,
+		TYPE_MAX,
+	};
```

Comment:
@Chaosus Can we rename this enum to something else? `Type` is super generic, and the field `Type type` produces warnings when generating the C# solution due to the generated `GetType` method.

Reply:
Of course, I didn't think about C# when committing this, sorry.